src/material-examples/typings.d.ts

```diff
@@ -0,0 +1 @@
+declare var module: {id: string};
```

Comment:
I see. Basically they shouldn't be necessary at all. It's fine to leave them for now though.

Just want to check. Does something need the moduleId in our development? (e.g Google internally)